### ISSUE TITLE
Tutorial links in the footer link to a wrong page 

### ISSUE THREAD

**Issue by OlivierJM** [rethinkdb/docs#1345]
**Describe the bug**
The tutorial links in the footer directs to a wrong for all languages displayed 

**To Reproduce**
1. Go to any page on https://rethinkdb.com2. 
2. Scroll down to the footer, under getting started click on tutorials 
3.  You will be redacted to a list of languages page here https://rethinkdb.com/docs/guide 
4. Click any of the languages displayed
5. Notice you will be redirected to a 404 page e.g: https://rethinkdb.com/docs/java/  

**Expected behavior**
It should redirect to a page with links that work like this one https://rethinkdb.com/docs/guide/javascript 

**Screenshots**

<img width="1436" alt="Screenshot 2022-08-24 at 9 30 27 PM" src="https://user-images.githubusercontent.com/11255454/186506578-878a9ae0-27f8-4ef1-a5e1-d02dead69ab4.png">

**System info**
 - OS: [e.g. macOS Mojave 10.14.3]
 - RethinkDB Version: [e.g. 2.4.0]

**Additional context**

**Comment by OlivierJM** [comment]
While trying to fix this issue, I went through the docs and www, I wasn't able to locate the footer.

**Comment by srh** [comment]
Thank you for trying to fix it and reporting it.  I have fixed it in #1346.

It's probably bad that the tutorial link points to `/docs/guides` instead of `/docs/guides/`, and I suppose it's bad that we don't have a redirect from `/docs/guides` to `/docs/guides/` and likewise for other index pages, so that we have a single canonical URL.

The footer is in `_layouts/default.html` in the www repo, and it was easier to find using `ack` than it would have been with `grep` because `ack` filters out the `_site` folder (as I found out).
